src/app/core/common-components/basic-autocomplete/basic-autocomplete.component.html

```diff
@@ -33,25 +33,60 @@
   autoActiveFirstOption
   [hideSingleSelectionIndicator]="multi"
 >
-  <mat-option
-    *ngFor="let item of autocompleteSuggestedOptions | async"
-    [value]="item"
+  <div *ngIf="!reorder">
```

Comment:
I think we could avoid copying the template block if we use `cdkDropListDisabled` (https://stackoverflow.com/a/54180096/1473411). Did you try that yet?